.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -39,24 +43,75 @@ If any further question about AME onboarding or validation tools, please view th
+    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits.
 
 - [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.
 
 - [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 
 
 ### Breaking Change Review Checklist 
-If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
-- [ ] Removing API(s) in a stable version
-- [ ] Removing properties in a stable version
-- [ ] Removing API version(s) in a stable version
-- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
-- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))
-
-**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+If you have any breaking changes as defined in the [Breaking Change Policy](http://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.
+  
+**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+  
+NOTE: To update API(s) in public preview for over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))
 
 Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).
+
+
+## Non-ARM API Information (Data Plane)
+
+This PR template is for Data Plane APIs only.
+
+### API Info: The Basics
+
+* Link to engagement record issue (if known): 
+
+Is this review for (select one):
+
+- [ ] a public preview
+- [ ] GA release 
+
```

Comment:
Should we also add a check list like below?

```
### REST API Review Checklist 

- [ ] Ensure you've reviewed [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). This is required before you can request review from Azure REST API Stewardship.
```

I see it is mentioned once in `ARM API Information` section and also included indirectivity in contact section. But we might want to emphasis the Guidelines again.